### PR TITLE
update fluid framework packets to latest

### PR DESCRIPTION
**Pull request by dannimad**

### CODE REVIEW

File: brainstorm/src/index.tsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ export async function start() {
         ({ container, services } = await client.getContainer(containerId, containerSchema));
     }
 
-    if (!container.connected) {
+    if (container.connectionState !== 2) {
```
Comment (dannimad) [inline brainstorm/src/index.tsx:44]: Not entirely sure if this is correct usage. If it is, I will change all other deprecated connected members to connectionState

Comment (dannimad) [reply brainstorm/src/index.tsx:44]: I can't import ConnectionState, that is why I am using its value.

Comment (scottn12) [reply brainstorm/src/index.tsx:44]: We should add `@fluidframework/container-loader` to the dependencies in package.json and import `ConnectionState` here.

File: angular-demo/package.json [hunk 2]
```diff
@@ -22,8 +22,8 @@
     "@angular/platform-browser": "~12.2.0",
     "@angular/platform-browser-dynamic": "~12.2.0",
     "@angular/router": "~12.2.0",
-    "@fluidframework/tinylicious-client": "^0.47.0",
-    "fluid-framework": "^0.47.0",
+    "@fluidframework/tinylicious-client": "^0.59.4000",
+    "fluid-framework": "^0.59.4000",
```
Comment (scottn12) [inline angular-demo/package.json:26]: @sonalivdeshpande informed me that 0.59.4000 is unstable and patch 0.59.4001 was released to fix the regressions found in 4000. However, 1.0.0 was also released, so we should just bump to 1.0.0 for the relevant packages

**Review by scottn12 — CHANGES_REQUESTED** [review]

**Comment by dannimad** [comment]
examples that use azure-client have to use 0.59.3000 version for compatibility issues. Rest of examples up to date to 1.0.0

File: brainstorm/package.json [hunk 3]
```diff
@@ -45,10 +45,11 @@
   },
   "dependencies": {
     "@fluentui/react": "^8.71.0",
-    "fluid-framework": "^0.48.1",
-    "@fluidframework/azure-client": "^0.48.1",
-    "@fluidframework/test-client-utils": "^0.48.1",
+    "@fluidframework/azure-client": "^0.59.3000",
+    "@fluidframework/container-loader": "^0.59.4001",
```
Comment (curtisman) [inline brainstorm/package.json:49]: should these be consistently update to 1.0.0?

File: brainstorm/src/index.tsx [hunk 4]
```diff
@@ -41,7 +42,7 @@ export async function start() {
         ({ container, services } = await client.getContainer(containerId, containerSchema));
     }
 
-    if (!container.connected) {
+    if (container.connectionState !== ConnectionState.Connected) {
```
Comment (curtisman) [inline brainstorm/src/index.tsx:45]: Shouldn't this be expose thru fluid-framework as well? @scottn12 I assume that these examples shouldn't import package individually?

File: brainstorm/package.json (hunk 3 above)
Comment (dannimad) [reply brainstorm/package.json:49]: That is what I would think but azure-client package seems outdated. Whenever I tried to update fluid-framework to 1.0.0, azure-client stayed behind and was causing incompatibility issues

**Review by curtisman — CHANGES_REQUESTED** [review]
:clock1:

File: angular-demo/package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluid-example/angular-demo",
-  "version": "0.1.0",
+  "version": "1.0.0",
```
Comment (skylerjokiel) [inline angular-demo/package.json:3]: No need to rev the version of these examples.

```suggestion
  "version": "0.1.0",
```

File: brainstorm/package.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluid-example/brainstorm",
-  "version": "0.48.1",
+  "version": "1.0.0",
```
Comment (skylerjokiel) [inline brainstorm/package.json:3]:
```suggestion
  "version": "0.1.0",
```

File: brainstorm/package.json (hunk 3 above)
Comment (skylerjokiel) [reply brainstorm/package.json:49]: azure-client is not yet released. These should be on the same version though.

```suggestion
    "@fluidframework/container-loader": "~0.59.3000",
```

File: brainstorm/src/index.tsx (hunk 4 above)
Comment (skylerjokiel) [reply brainstorm/src/index.tsx:45]: This was a miss. We don't want azure developers pulling in other packages. @scottn12 I created [Bug 716](https://dev.azure.com/fluidframework/internal/_workitems/edit/716): Re-export ConnectionState from fluid-framework. Let's get this into the next minor release.

In the mean time...

```suggestion
    // TODO: Export ConnectionState type https://github.com/microsoft/FluidFramework/issues/10681
    if (container.connectionState !== 2 /*Connected*/) {
```

File: collaborative-text-area/package.json [hunk 7]
```diff
@@ -1,21 +1,22 @@
...
   "homepage": "https://fluidframework.com",
   "repository": "microsoft/FluidExamples",
   "license": "MIT",
   "author": "Microsoft and contributors",
   "private": true,
   "dependencies": {
-    "@fluid-experimental/react-inputs": "^0.50.1",
-    "@fluidframework/tinylicious-client": "^0.50.1",
+    "@fluid-experimental/react-inputs": "^1.0.0",
+    "@fluidframework/container-loader": "^1.0.0",
```
Comment (skylerjokiel) [inline collaborative-text-area/package.json:12]: don't pull this in.

```suggestion
```

File: collaborative-text-area/src/App.tsx [hunk 8]
```diff
@@ -29,7 +30,7 @@ const useSharedString = (): SharedString => {
     }
     else {
       container = (await client.getContainer(containerId, containerSchema)).container;
-      if (!container.connected) {
+      if (container.connectionState !== ConnectionState.Connected) {
```
Comment (skylerjokiel) [inline collaborative-text-area/src/App.tsx:33]:
```suggestion
      // TODO: Export ConnectionState type https://github.com/microsoft/FluidFramework/issues/10681
      if (container.connectionState !== 2 /*Connected*/) {
```

File: multi-framework-diceroller/package.json [hunk 9]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@fluid-example/multi-framework-diceroller",
-    "version": "0.48.1",
+    "version": "1.0.0",
```
Comment (skylerjokiel) [inline multi-framework-diceroller/package.json:3]:
```suggestion
    "version": "0.1.0",
```

File: node-demo/package.json [hunk 10]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluid-example/node-demo",
-  "version": "0.1.0",
+  "version": "1.0.0",
```
Comment (skylerjokiel) [inline node-demo/package.json:3]:
```suggestion
  "version": "0.1.0",
```

File: react-demo/package.json [hunk 11]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluid-example/react-demo",
-  "version": "0.48.1",
+  "version": "1.0.0",
```
Comment (skylerjokiel) [inline react-demo/package.json:3]:
```suggestion
  "version": "0.1.0",
```

File: react-starter-template/package.json [hunk 12]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-starter-template",
-  "version": "0.48.1",
+  "version": "1.0.0",
```
Comment (skylerjokiel) [inline react-starter-template/package.json:3]:
```suggestion
  "version": "0.1.0",
```

File: teams-fluid-hello-world/package.json [hunk 13]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "teams-fluid-hello-world",
-  "version": "0.48.1",
+  "version": "1.0.0",
```
Comment (skylerjokiel) [inline teams-fluid-hello-world/package.json:3]:
```suggestion
  "version": "0.1.0",
```

**Review by skylerjokiel — CHANGES_REQUESTED** [review]

File: brainstorm/src/index.tsx (hunk 4 above)
Comment (curtisman) [reply brainstorm/src/index.tsx:45]: @skylerjokiel can we add a policy check to make sure we don't accidentally import `inner packages` in these examples?

Comment (curtisman) [reply brainstorm/src/index.tsx:45]: AB#718

File: brainstorm/package.json (hunk 3 above)
Comment (dannimad) [reply brainstorm/package.json:49]: removing container-loader

**Review by curtisman — APPROVED** [review]
:shipit:

**Pull request merged**